src/bitcoind.cpp

```diff
@@ -115,7 +115,7 @@ bool AppInit(int argc, char* argv[])
                    "depending on how you installed Zcash:\n") +
                  _("- Source code:  %s\n"
                    "- .deb package: %s\n")).c_str(),
-                GetConfigFile().string().c_str(),
+                GetConfigFile(GetArg("-conf", BITCOIN_CONF_FILENAME)).string().c_str(),
                 "contrib/debian/examples/zcash.conf",
                 "/usr/share/doc/zcash/examples/zcash.conf");
```

Comment:
If someone forked zcashd and changed ``BITCOIN_CONF_FILENAME``, they might miss these.